npm-shrinkwrap.json

```diff
@@ -5191,794 +5190,6 @@
       "version": "1.0.0",
       "integrity": "sha1-FQStJSMVjKpA20onh8sBQRmU6k8="
     },
-    "fsevents": {
```

Comment:
Note that fsevents is an optional dependency. So I suppose we'll see thrashing, whenever someone using OSX vs anything else updates the shrinkwrap. https://github.com/npm/npm/issues/17722